core/lexicon/en/setting.inc.php

```diff
@@ -471,6 +471,9 @@
 $_lang['setting_new_folder_permissions'] = 'New Folder Permissions';
 $_lang['setting_new_folder_permissions_desc'] = 'When creating a new folder in the File Manager, the File Manager will attempt to change the folder permissions to those entered in this setting. This may not work on some setups, such as IIS, in which case you will need to manually change the permissions.';
 
+$_lang['setting_package_installer_at_top'] = 'Pin Package-Installer at top';
+$_lang['setting_package_installer_at_top_desc'] = 'If enabled, the package-installer menu-entry inside the Extras-menu will be pinned at the top - regardless of its "menuindex"-attribute. If disabled, the package-installer will be positioned according to its menuindex.';
```

Comment:
There are a lot of unnecessary dashes in the description. Also suggesting other tweaks to keep it concise.
```suggestion
$_lang['setting_package_installer_at_top_desc'] = 'If enabled, the Installer entry inside the Extras menu will be pinned at the top. Otherwise it will be positioned according to its menuindex.';
```